src/controllers/parcelController.js

```diff
@@ -1,7 +1,7 @@
 import { randomBytes } from 'crypto';
 import parcels from '../db/parcel';
 
-export default {
+class Reflection {
```

Comment:
Your class should mirror the name of the file in which it resides. In this case your class should be `PercelController`

Reply:
Okay.